### PR TITLE
DDFFORM 827 simple linklist

### PR DESCRIPTION
**Pull request by LasseStaus**
- **Create paragraph `simple links` which uses an unlimited amount of field_links.**
- **Create paragraph twig template `paragraph--simple-links.html.twig`**

#### Link to issue

jira: https://reload.atlassian.net/browse/DDFFORM-827
https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/1369
design system: https://github.com/danskernesdigitalebibliotek/dpl-design-system/pull/683

#### Description

Create a paragraph "simple links" 

#### Screenshot of the result

image

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -56,9 +56,9 @@
             "package": {
                 "name": "danskernesdigitalebibliotek/dpl-design-system",
                 "type": "drupal-library",
-                "version": "2024.28.0",
+                "version": "2024.28.1",
```
Comment (kasperbirch1) [inline composer.json:59]: Remember to change this after the design system is merged

File: config/sync/core.entity_view_display.paragraph.simple_links.default.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    - paragraphs.paragraphs_type.simple_links
+  module:
+    - link
+id: paragraph.simple_links.default
+targetEntityType: paragraph
+bundle: simple_links
+mode: default
+content:
+  field_link:
+    type: link
```
Comment (kasperbirch1) [inline config/sync/core.entity_view_display.paragraph.simple_links.default.yml:16]: Should this be `linkit`?

File: web/themes/custom/novel/templates/paragraphs/paragraph--simple-links.html.twig [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (kasperbirch1) [inline web/themes/custom/novel/templates/paragraphs/paragraph--simple-links.html.twig:1]: Remove this line

**Review by kasperbirch1 — APPROVED** [review]
Looks good! Do you want to add it to default content too? 😄

File: config/sync/core.entity_view_display.paragraph.simple_links.default.yml (hunk 2 above)
Comment (LasseStaus) [reply config/sync/core.entity_view_display.paragraph.simple_links.default.yml:16]: Good catch, but this is the `entity_view_display`and not the `entity_form_display`. In our case i think we only to ensure the field is display'ed as linkit to the editors.

**Pull request merged**